core/src/test/java/apoc/create/CreateTest.java

```diff
@@ -415,6 +415,35 @@ public void testVirtualFromNodeFunction() {
                 });
     }
 
+    @Test
+    public void test2VirtualFromNodesHaveUniqueIDs() {
```

Comment:
Would

```
CREATE (n1:Cat { name:'Maja', born: 2023 } )
WITH n1
UNWIND [1,2,3] AS i
RETURN apoc.create.virtual.fromNode(n1, ['name']) AS node1
```

be a useful test? I would expect it to return three new vnodes with the same property and label but different ids.

Maybe such test already exist. I have looked at the whole class.

Reply:
Ah yes, that is useful and was what I was meant to test 😅 not sure why I wrote the test I did